include/triton/Dialect/TritonGPU/IR/LinearLayoutConversions.h

```diff
@@ -37,12 +37,18 @@ namespace mlir::triton::gpu {
 // to compute the linear layout for MMAv3 (i.e. Hopper) shared layouts (i.e.
 // shared layouts with hasLeadingOffset == true) but is otherwise unused.
 //
+// inThreadTranspose is a flag indicating if transpose should be performed while
+// the data resides in thread-local registers. This is set to true on AMD
+// platform when K-major matrix is about to be written into LDS (shared memory)
```

Comment:
I think we agree to use KContig to indicate k-minor. So here you may want to use non-KContig.